reconcile/statuspage/page.py

```diff
@@ -91,3 +92,23 @@ def init_from_page(
                 for c in page.components or []
             ],
         )
+
+
+class StatusMaintenace(BaseModel):
```

Comment:
very important typo!

```suggestion
class StatusMaintenance(BaseModel):
```

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4344/commits/3e6ac6ac984645f8215b0394c32e37082c43f8ba